NOTES.md

```diff
@@ -2,6 +2,8 @@
 
 ## Ready For Dev
 
+* Fix choam charity to not be claimable after auction has begun
```

Comment:
I was thinking about this. I think the duration of that period is too short unless there is something to pause the game which seems unnecessary. I think it's better if you can just claim it whenever

Reply:
Oh!  but what happened in one my test runs was, I went into the round with a tone of money but then I spent all my money on the auction and then got charity after that!